VideoApi/VideoApi.Contract/Consts/QuickLinkParticipantConst.cs

```diff
@@ -0,0 +1,8 @@
+namespace VideoApi.Contract.Consts
```

Comment:
could we move this const from the contract, or duplicate it in the domain, because we dont want to add contract reference to the DAL layer

Reply:
the DAL layer already has reference to VideoApi.Common, so might be worth moving this const into common instead